source/reverse-sync.txt

```diff
@@ -0,0 +1,189 @@
+.. _c2c-reverse-process:
+
+======================
+Reverse Sync Direction
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :class: singlecol
+   :depth: 1
+
+To reverse the direction of a sync operation, use the following
+procedure to commit your sync and call the ``reverse`` endpoint. 
+
+Before You Begin
+----------------
+
+Before you can reverse your sync direction, you must configure
+``mongosync`` and use the :ref:`c2c-api-start` endpoint to set the
+following parameters:
+
+- ``reversible`` to ``true``
+- ``enableUserWriteBlocking`` to ``true``.
+
+For more information on limitations and requirements of reversing sync,
+see :ref:`c2c-api-reverse`. 
+
+Steps
+-----
+
+Use these steps to reverse the direction of your sync:
+
+.. procedure::
+   :style: normal
+
+   .. step:: Verify the status of the ``mongosync`` process.
+
+      Call the :ref:`progress <c2c-api-progress>` endpoint to determine
+      the status of ``mongosync`` before starting the cutover process.
+      Ensure that the ``mongosync`` process status indicates the
+      following values:
+
+      - ``canCommit`` is ``true``.
+
+      - ``lagTimeSeconds`` is small (near ``0``).
+
+        If ``lagTimeSeconds`` isn't close to ``0`` when the cutover
+        starts, cutover might take a long time.
+
+      The following example returns the status of the synchronization process.
+
+      Request
+      ~~~~~~~
+
+      .. literalinclude:: /includes/api/requests/progress.sh
+         :language: shell
+
+      Response
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/responses/progress.json
+         :language: json
+         :copyable: false
+         :emphasize-lines: 5, 8
+
+   .. step:: Stop any write operations to the synced collections on the source.
+
+      - If you started ``mongosync`` with ``enableUserWriteBlocking``
```

Comment:
[tense]

Update to present tense. "If you start...".

Reply:
Done!